src/canvas-extractor/.vscode/settings.json

```diff
@@ -99,5 +99,6 @@
         "**/__pycache__": true,
         "**/*.egg-info": true
     },
-    "jupyter.jupyterServerType": "local"
+    "jupyter.jupyterServerType": "local",
+    "python.pythonPath": "c:\\work\\fizz\\Ed-Fi-X-Fizz\\src\\canvas-extractor\\.venv\\Scripts\\python.exe"
```

Comment:
Is it possible to make that relative instead of absolute, so that it will work for other developers?

Reply:
Ouch, yes.  Was generated by vscode